Donut/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
+
+from Donut import config, constants
+from Donut.modules import example
+
+app = Flask(__name__)
+app.debug = False
+app.secret_key = config.SECRET_KEY
+
+# Maximum file upload size, in bytes.
+app.config['MAX_CONTENT_LENGTH'] = constants.MAX_CONTENT_LENGTH
+
+# Load blueprint modules
+app.register_blueprint(example.blueprint, url_prefix='/example')
+
+# Create database engine object.
+# TODO##DatabaseWork: We currently don't have a database set up, so we can't
+# reference sqlalchemy yet. However, it serves as a good example implementation.
+# engine = create_engine(config.DB_URI, convert_unicode=True)
+
+@app.before_request
+def before_request():
+  """Logic executed before request is processed."""
```

Comment:
:+1: I looked up the style guide for docstrings and this looks good. I think we should stick with these styles:
```
"""This is a short docstring."""
```
or
```
"""
This is a multiple line
docstring that spans
additional lines.
"""
```
The one I think looks really weird is this (let's not use it):
```
"""This is a
multi line docstring.
"""